cabal-install/src/Distribution/Client/Init/Types.hs

```diff
@@ -455,7 +455,12 @@ newtype BreakException = BreakException String deriving (Eq, Show)
 instance Exception BreakException
 
 -- | Used to inform the intent of prompted messages.
-data Severity = Info | Warning | Error deriving (Eq, Show)
+data Severity = Info | Warning | Error deriving (Eq)
+
+displaySeverity :: Severity -> String
```

Comment:
I dunno where is a good place to put this function.

Reply:
it's fine under it like that. Usually i prefer grouping functions with the relevant data types if it's a simple choice to make. More complex functionality is harder.